content/usage/advanced/index.md

```diff
@@ -284,94 +538,197 @@ being redirected via BlueOS, but can also be used to display online videos if th
 
-#### Mini Widgets
-
-Mini widgets are small, generally single-function widgets that can be drag-positioned in the
-[header bar](#header-bar), footer bar, or any [mini widget bar](#mini-widget-bar).
-
-They are editable by selecting "Mini Widgets" in the bottom left corner of edit mode, then either
-dragging a new mini-widget (from those available along the bottom of the screen) into a
-[mini-widget bar](#mini-widget-bar), or configuring or removing one from the "current mini-widgets"
-list in the bottom left corner.
-
-The current options include
-- Arm/Disarm toggle switch
-- Vehicle connection status indicator
-- Power / battery indicator
-- Depth indicator
-- (Relative) altitude indicator
-- Very generic indicator
-    - configuring this allows selecting which vehicle variable to track, out of
-      any that have been received so far (including custom ones)
-    - only variables coming from Ardupilot vehicles are currently supported
-    - available variables include those comming from `NAMED_VALUE_FLOAT/INT` messages as well
-      as any variable that is inside any MAVLink message
-    - several pre-made presets are available for usage with common variables
-    - it is also possible to specify a display unit, a value multiplier, an icon, the number of
-      digits after the decimal place and a custom display name
-{{ easy_image(src="very-generic-widget-config-presets", width=300, center=true) }}
-{{ easy_image(src="very-generic-widget-config-custom", width=300, center=true) }}
-- Video recorder
-    - allows recording one of the available WebRTC streams, or the full Cockpit tab
-    - recording occurs in the browser of the display device (not onboard the vehicle)
-        - this is currently stored in memory and downloaded to the device when finished, which may limit
-        maximum time for individual recordings
-    - recordings are saved using the [mission name](#mission-name) and the starting timestamp
-    - a warning is displayed if Cockpit is closed while a video is recording, and a recovery popup
-    appears when Cockpit is next opened in that browser / on that device
-{{ easy_image(src="video-recording-config", width=250, center=true) }}
-{{ easy_image(src="video-recording-termination-warning", width=400, center=true) }}
-{{ easy_image(src="video-recording-recovery", width=400, center=true) }}
-- Joystick connection status indicator
-- Flight mode selector
-- GPS status indicator
-- [View](#views) selector
-- Takeoff/land button
+##### Custom Widget Base
+
+{{ easy_image(src="custom-widget-base", width=250, center=true) }}
+
+#### Input Widgets
+
+Input widgets should be placed in a [Custom Widget Base](#custom-widget-base), and can be configured by
+clicking on them when in [Edit Mode](#edit-mode).
+
+While other widgets generally have predefined behaviour, these are specifically designed to trigger and
+set variables for use as parameters in [Custom HTTP Actions](#custom-actions).
```

Comment:
They actually set variables in the data-lake, and so can be used in any Actions (or other places like external widgets, DIY widgets, etc).